apps/SageAccountWeb/src/components/LeftRightPanel.tsx

```diff
@@ -6,8 +7,8 @@ export function LeftRightPanel({
   rightContent,
   className,
 }: {
-  leftContent: JSX.Element
-  rightContent: JSX.Element
```

Comment:
The ambient `JSX` type was removed. We could `import { JSX } from 'react'` but in all of these cases, `ReactNode` is a better type to use.